### PR TITLE
contracts-bedrock: mainnet-deployment

### PR DESCRIPTION
**Pull request by tynes**
**Description**

Deploy config updates for mainnet contract deployments

### CODE REVIEW

File: packages/contracts-bedrock/deploy-config/mainnet.json [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
-  "l2OutputOracleStartingBlockNumber": 79149704,
-  "l2OutputOracleProposer": "0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC",
-  "l2OutputOracleChallenger": "0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC",
-  "finalizationPeriodSeconds": 2,
+  "l2OutputOracleSubmissionInterval": 1800,
+  "l2OutputOracleStartingTimestamp": 0,
+  "l2OutputOracleStartingBlockNumber": 0,
+  "l2OutputOracleProposer": "0x473300df21D047806A082244b417f96b32f13A33",
+  "l2OutputOracleChallenger": "0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A",
+  "finalizationPeriodSeconds": 604800,
```
Comment (K-Ho) [inline packages/contracts-bedrock/deploy-config/mainnet.json:20]: 1 week 👍

File: packages/contracts-bedrock/deploy-config/mainnet.json [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
-  "l2OutputOracleSubmissionInterval": 20,
-  "l2OutputOracleStartingTimestamp": 1679069195,
-  "l2OutputOracleStartingBlockNumber": 79149704,
-  "l2OutputOracleProposer": "0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC",
-  "l2OutputOracleChallenger": "0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC",
-  "finalizationPeriodSeconds": 2,
+  "l2OutputOracleSubmissionInterval": 1800,
+  "l2OutputOracleStartingTimestamp": 0,
+  "l2OutputOracleStartingBlockNumber": 0,
+  "l2OutputOracleProposer": "0x473300df21D047806A082244b417f96b32f13A33",
```
Comment (K-Ho) [inline packages/contracts-bedrock/deploy-config/mainnet.json:18]: lgtm: https://etherscan.io/address/0x473300df21D047806A082244b417f96b32f13A33

File: packages/contracts-bedrock/deploy-config/mainnet.json [hunk 3]
```diff
@@ -28,8 +28,8 @@
   "l2GenesisBlockGasLimit": "0x1c9c380",
   "l2GenesisBlockCoinbase": "0x4200000000000000000000000000000000000011",
   "l2GenesisBlockBaseFeePerGas": "0x3b9aca00",
-  "gasPriceOracleOverhead": 2100,
-  "gasPriceOracleScalar": 1000000,
+  "gasPriceOracleOverhead": 188,
```
Comment (K-Ho) [inline packages/contracts-bedrock/deploy-config/mainnet.json:31]: looks good based on: https://docs.google.com/spreadsheets/u/1/d/1lUsaMUNDHEXpLsnbGUeDFINb8xIu6jOAFUnni7q9BCo/edit#gid=334764629

File: packages/contracts-bedrock/deploy-config/mainnet.json [hunk 4]
```diff
@@ -28,8 +28,8 @@
...
+  "gasPriceOracleScalar": 684000,
   "eip1559Denominator": 50,
   "eip1559Elasticity": 10,
```
Comment (K-Ho) [inline packages/contracts-bedrock/deploy-config/mainnet.json]: Should be 6: see https://www.notion.so/oplabs/Optimism-Gas-Target-1559-Params-for-Mainnet-Launch-864e8f41e68f4743b74ac029b67fbeef

File: packages/contracts-bedrock/deploy-config/mainnet.json [hunk 5]
```diff
@@ -1,23 +1,23 @@
...
   "l1ChainID": 1,
   "l2ChainID": 10,
   "l2BlockTime": 2,
   "maxSequencerDrift": 600,
   "sequencerWindowSize": 3600,
   "channelTimeout": 300,
-  "p2pSequencerAddress": "0x15d34AAf54267DB7D7c367839AAf71A00a2C6A65",
+  "p2pSequencerAddress": "0xAAAA45d9549EDA09E70937013520214382Ffc4A2",
   "batchInboxAddress": "0xff00000000000000000000000000000000000010",
   "batchSenderAddress": "0x70997970C51812dc3A010C7d01b50e0d17dc79C8",
```
Comment (K-Ho) [inline packages/contracts-bedrock/deploy-config/mainnet.json:14]: Shouldn't this be: https://etherscan.io/address/0x6887246668a3b87f54deb3b94ba47a6f63f32985

**Review by K-Ho — COMMENTED** [review]
couple values need tweaking

File: packages/contracts-bedrock/deploy-config/mainnet.json [hunk 6]
```diff
@@ -1,23 +1,23 @@
...
+  "batchSenderAddress": "0x6887246668a3b87F54DeB3b94Ba47a6f63F32985",
+  "l2OutputOracleSubmissionInterval": 1800,
+  "l2OutputOracleStartingTimestamp": 0,
+  "l2OutputOracleStartingBlockNumber": 0,
+  "l2OutputOracleProposer": "0x473300df21D047806A082244b417f96b32f13A33",
+  "l2OutputOracleChallenger": "0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A",
+  "finalizationPeriodSeconds": 604800,
   "proxyAdminOwner": "0x90F79bf6EB2c4f870365E785982E1f101E93b906",
   "baseFeeVaultRecipient": "0xa3d596EAfaB6B13Ab18D40FaE1A962700C84ADEa",
   "l1FeeVaultRecipient": "0xa3d596EAfaB6B13Ab18D40FaE1A962700C84ADEa",
```
Comment (K-Ho) [inline packages/contracts-bedrock/deploy-config/mainnet.json:23]: Do we know these values now? should we zero them out otherwise in the meantime?

**Review by maurelian — APPROVED** [review]

**Pull request merged**